bin/crew

```diff
@@ -1492,35 +1492,30 @@ def print_deps_tree(args)
   puts tree_view
 end
 
-def upload(pkg_name = nil, pkg_version = nil)
+def upload(pkg_name = nil, pkg_version = nil, gitlab_token = nil, binary_compression = nil)
   abort "\nPackage to be uploaded was not specified.\n".lightred if pkg_name.nil?
-  abort "\nGITLAB_TOKEN environment variable not set.\n".lightred if ENV.fetch('GITLAB_TOKEN', nil).nil?
+  abort "\nGITLAB_TOKEN environment variable not set.\n".lightred if gitlab_token.nil?
 
   packages = pkg_name
-  binary_compression_set = nil
-  gitlab_token = ENV.fetch('GITLAB_TOKEN', nil)
-  base_url = 'https://gitlab.com/api/v4/projects/26210301/packages/generic'
 
-  if pkg_name.nil?
-    %w[.tar.xz .tar.zst].each do |ext|
-      packages += `find #{CREW_LOCAL_REPO_ROOT}/release/*/*#{ext} -exec basename -s #{ext} {} + | cut -d- -f1 | LC_ALL=C sort | uniq | xargs`
-      packages += ' '
-    end
-    abort 'No package binaries found.'.lightred if packages.empty?
-  end
```

Comment:
Why are we removing this logic?  It would be helpful to upload in bulk for multiple binaries, right?

Reply:
There is already upload being invoked for each package. There should be one upload per package per architecture...